mypy/checkexpr.py

```diff
@@ -2087,16 +2087,6 @@ def analyze_ordinary_member_access(self, e: MemberExpr,
 
             return member_type
 
-    def analyze_external_member_access(self, member: str, base_type: Type,
```

Comment:
Hmm removing this adds some boilerplate to call sites. I have an alternative fix: rename this to `analyze_member_access`, `analyze_get_member` or `analyze_simple_member_access` (or similar) and remove the docstring and TODO comment (and possibly write a better docstring that explains the new purpose of this).

Reply:
Ok! 👍